docs/api/ref/requestBodies/add_or_edit_a_product.yaml

```diff
@@ -16,33 +16,24 @@ properties:
     example: mypassword
   comment:
     type: string
-    description: A comment for the change. It will be shown in product changes history.
+    description: A comment for the change. It will be shown in product changes history.
     example: new packaging from super-app
   brands:
-    schema:
-      type: array
-      items:
-        type: string
-    style: form
-    explode: false
```

Comment:
this is normally important as it tells that you have to use `brands=lindt,lipton` and not `brands=lindt&brands=lipton`
see https://spec.openapis.org/oas/latest.html#style-values

Reply:
Hum, ok those are in a body, so it's not a schema, so your changes are right.